### PR TITLE
Add codesandbox

### PR DESCRIPTION
**Pull request by buoyantair**
Codesandbox is opensource for the past few months now.... It's a great resource to learn from!

### CODE REVIEW

**Comment by johnfaucett** [comment]
Hey @buoyantair, this looks like a nice project. I would be happy to merge if you could address the few comments I made.

File: README.md [hunk 1]
```diff
@@ -18,6 +18,13 @@ This list is for developers who are looking for non-trivial quality applications
 ## Web Apps (frontend only)
 ----
 
+### [Codesandbox](https://github.com/CompuIves/codesandbox-client)
+
+* react, ngix, phoenix
```
Comment (johnfaucett) [inline README.md]: Do you mean `nginx`? Or is ngix some js package or tool I'm not aware of?

Comment (buoyantair) [reply README.md]: nginx. Sorry I will fix this.

**Comment by johnfaucett** [comment]
@buoyantair Thanks for helping out finding awesome foss apps to learn from.

File: README.md (hunk 1 above)
Comment (jojxl0l) [reply README.md]: @mickael-kerjean 010001

**Pull request merged**